array_api_tests/pytest_helpers.py

```diff
@@ -33,3 +38,22 @@ def doesnt_raise(function, message=''):
 
 def nargs(func_name):
     return len(getfullargspec(getattr(function_stubs, func_name)).args)
+
+
+def assert_dtype(
+    func_name: str,
+    in_dtypes: Tuple[DataType, ...],
+    out_name: str,
+    out_dtype: DataType,
+    expected: DataType
```

Comment:
This function feels like it has too many arguments. `expected` could just default to taking the result type of the in_dtypes. Is the out_name really needed either?